specs/deliverly_server_SUITE.erl

```diff
@@ -85,6 +94,56 @@ auth_failed(_) ->
   1 = Size,
   ok.
 
+auth_token_success(_) ->
+  Token = proplists:get_value(token, de_auth_token:request_token(#{})),
+  Res = deliverly_server:auth_client(#de_client{socket=1, app = default}, [{<<"token">>, Token}]),
+  {reply, _, _} = Res,
+  ok.
+
+auth_token_failed(_) ->
+  Res = deliverly_server:auth_client(#de_client{socket=1, app = default}, [{<<"token">>, <<"123">>}]),
+  {error, 3401} = Res,
+  ok.
+
+auth_token_timeout(_) ->
+  Token = proplists:get_value(token, de_auth_token:request_token(#{expires_in => 1})),
```

Comment:
Line 109 is too long:   Token = proplists:get_value(token, de_auth_token:request_token(#{expires_in => 1})),.